examples/manifold/plot_manifold_sphere.py

```diff
@@ -30,17 +30,15 @@
 # License: BSD 3 clause
 
 from time import time
-
 import numpy as np
 import matplotlib.pyplot as plt
-from mpl_toolkits.mplot3d import Axes3D
 from matplotlib.ticker import NullFormatter
-
 from sklearn import manifold
 from sklearn.utils import check_random_state
 
-# Next line to silence pyflakes.
-Axes3D
+# Unused but required import for doing 3d projections with matplotlib < 3.2
+import mpl_toolkits.mplot3d  # noqa: F401
+import warnings  # TODO(1.2) Remove.
```

Comment:
I prefer to have `# TODO` comment near `warnings.filterwarnings`, so there is some context about what is being removed.

Once the `warnings.catch_warning` code is removed, the linter should catch this import as unused and error.